lib/app_funcs.sh

```diff
@@ -60,6 +60,17 @@ function compile_app() {
   cd - > /dev/null
 }
 
+function post_compile_hook() {
+  cd $build_path
+
+  if [ -f post_compile.sh ]; then
+    output_section "Executing post_compile.sh"
+    ./post_compile.sh || exit 1
```

Comment:
With the new multi-buildpack feature, I assume buildpacks that are run before this buildpack, will make their languages/tools available too? This could be `post_compile` instead. That way we could allow the user to set the interpreter line to anything that's available on Heroku.